src/opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQSet.cpp

```diff
@@ -38,7 +39,7 @@ bool UDQScalar::defined() const {
 
 double UDQScalar::value() const {
     if (!this->m_defined)
-        throw std::invalid_argument("UDQSCalar: Value not defined");
+        throw std::invalid_argument("UDQSCalar: Value not defined  wgname:" + this->m_wgname);
```

Comment:
Maybe add a space after the `':'` here to avoid a string like
```
... not defined  wgname:W_1
```
:question:

Reply:
Fixed